.rubocop_todo.yml

```diff
@@ -0,0 +1,13 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-10-24 20:49:44 -0500 using RuboCop version 0.50.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 34
+# Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
+# URISchemes: http, https
+Metrics/LineLength:
+  Max: 117
```

Comment:
117 Seems like a pretty specific number :) Why not a round number like 90/100/110/120?

Reply:
117 is definitely specific, though not derived from me. ;) When you run rubocop's`--auto-gen-config` command it pushes all violations into the todo file so that subsequent runs of rubocop will pass. For the case of the LineLength cop, it sets the number to the current longest line in the files it's checked.

I'd like to get it to a lower, rounder number, but that'll take some work and discussion. I'm leaving it be for now as that's not specific to the Rails work, needs more work, etc.